lib/renderer/security-warnings.ts

```diff
@@ -103,10 +103,14 @@ const warnAboutInsecureResources = function () {
     return;
   }
 
+  const isLocal = (url: URL): boolean =>
+    ['localhost', '127.0.0.1', '[::1]', ''].includes(url.hostname);
+  const isInsecure = (url: URL): boolean =>
+    ['http:', 'ftp:'].includes(url.protocol) && !isLocal(url);
```

Comment:
`ftp` urls will no longer resolve in main since https://github.com/electron/electron/pull/30503 :)

https://bugs.chromium.org/p/chromium/issues/detail?id=333943